contrib/keycloak/docker-compose.dev-keycloak.yml

```diff
@@ -16,7 +16,7 @@ services:
   elasticsearch:
     build:
       context: services/elasticsearch
-    image: alephdata/aleph-elasticsearch:${ALEPH_TAG:-latest}
+    image: ghcr.io/alephdata/aleph-elasticsearch:${ALEPH_TAG:-latest}
```

Comment:
I think we might need to pin this to a commit hash tag (because we don’t have versioned releases for this image)?

Reply:
Yeah, I think that we should perhaps get rid of latest and pin to a version. Might even be something that we can automate in time (we don't need to do that now though)